patches/dompurify+2.3.4.patch

```diff
@@ -0,0 +1,52 @@
+diff --git a/node_modules/dompurify/dist/purify.js b/node_modules/dompurify/dist/purify.js
+index 68bc40f..03bd803 100644
+--- a/node_modules/dompurify/dist/purify.js
++++ b/node_modules/dompurify/dist/purify.js
+@@ -623,6 +623,12 @@
+ 
+     var HTML_INTEGRATION_POINTS = addToSet({}, ['foreignobject', 'desc', 'title', 'annotation-xml']);
+ 
++    // Certain elements are allowed in both SVG and HTML
++    // namespace. We need to specify them explicitly
++    // so that they don't get erronously deleted from
++    // HTML namespace.
++    var COMMON_SVG_AND_HTML_ELEMENTS = addToSet({}, ['title', 'style', 'font', 'a', 'script']);
++
+     /* Keep track of all possible SVG and MathML tags
+      * so that we can perform the namespace checks
+      * correctly. */
+@@ -707,15 +713,9 @@
+           return false;
+         }
+ 
+-        // Certain elements are allowed in both SVG and HTML
+-        // namespace. We need to specify them explicitly
+-        // so that they don't get erronously deleted from
+-        // HTML namespace.
+-        var commonSvgAndHTMLElements = addToSet({}, ['title', 'style', 'font', 'a', 'script']);
+-
+         // We disallow tags that are specific for MathML
+         // or SVG and should never appear in HTML namespace
+-        return !ALL_MATHML_TAGS[tagName] && (commonSvgAndHTMLElements[tagName] || !ALL_SVG_TAGS[tagName]);
++        return !ALL_MATHML_TAGS[tagName] && (COMMON_SVG_AND_HTML_ELEMENTS[tagName] || !ALL_SVG_TAGS[tagName]);
+       }
+ 
+       // The code should never reach this place (this means
+@@ -908,7 +908,7 @@
+       }
+ 
+       /* Check if tagname contains Unicode */
+-      if (stringMatch(currentNode.nodeName, /[\u0080-\uFFFF]/)) {
++      if (regExpTest(/[\u0080-\uFFFF]/, currentNode.nodeName)) {
```

Comment:
This likely makes very little difference, but `Regex.text` is faster than `String.match`. There are even eslint rules for this normally, see: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-regexp-test.md